### PR TITLE
Allow slash in vhost name #37

### PR DESCRIPTION
**Pull request by romainjanvier**
Just replace slash by %2F.
Tested manually.

### CODE REVIEW

File: src/Bab/RabbitMq/VhostManager.php [hunk 1]
```diff
@@ -18,10 +18,7 @@ class VhostManager
     public function __construct(array $credentials, Action $action, HttpClient $httpClient)
     {
         $this->credentials = $credentials;
-        if ('/' === $this->credentials['vhost']) {
-            $this->credentials['vhost'] = '%2f';
-        }
-
+        $this->credentials['vhost'] = str_replace('/', '%2F', $this->credentials['vhost']);
```
Comment (odolbeau) [inline src/Bab/RabbitMq/VhostManager.php]: Is there a reason to use `%2F` instead of `%2f`?

Comment (romainjanvier) [reply src/Bab/RabbitMq/VhostManager.php]: ... no. Do you prefer lowercase ?

Comment (odolbeau) [reply src/Bab/RabbitMq/VhostManager.php]: As this is what is used is the [official documentation](https://www.rabbitmq.com/uri-spec.html) yes please. :)

**Comment by odolbeau** [comment]
Thanks!

**Pull request merged**